content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Assign Pods to Nodes using Node Affinity
+content_template: templates/task
+weight: 120
+---
+
+{{% capture overview %}}
+This page shows how to assign a Kubernetes Pod to a particular node using Node Affinity in a
+Kubernetes cluster.
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
When using `{{< version-check >}}`, make sure to set `min-kubernetes-server-version` in the front matter metadata.

Reply:
@sftim , I am not sure hot to add `min-kubernetes-server-version` Can you please point me to exisitng page where this is used. Or can I remove `{{< version-check >}} if it is not required.`

Reply:
Have a look at line 3 of https://raw.githubusercontent.com/kubernetes/website/23a28d6dd288293d6d2970a9381f48753b7969fe/content/en/docs/tasks/configure-pod-container/share-process-namespace.md#L3